exercises/practice/robot-simulator/.meta/config.json

```diff
@@ -1,6 +1,35 @@
 {
   "blurb": "Write a robot simulator.",
   "authors": [],
```

Comment:
Another empty one, added in https://github.com/exercism/elixir/commit/ad9029641e5b09315979d125b01ce3c816229877 by Chris Doggett